Packages/com.llealloo.audiolink/Editor/Scripts/AudioLinkPipelineDetector.cs

```diff
@@ -0,0 +1,24 @@
+using UnityEngine;
+using UnityEditor;
+
+namespace AudioLink.Editor
+{
+    public static class AudioLinkPipelineDetector
+    {
+        [InitializeOnLoadMethod]
+        private static void DetectPackages()
+        {
+#if URP_AVAILABLE
+        Shader.EnableKeyword("UNIVERSAL_RENDER_PIPELINE");
```

Comment:
some weird indentation going on here

Reply:
When rider refuses to autoformat code in disabled compiler directives. Fixed.